Index: MyShelfie/src/main/java/it/polimi/ingsw/model/Shelf.java
===================================================================
diff --git a/MyShelfie/src/main/java/it/polimi/ingsw/model/Shelf.java b/MyShelfie/src/main/java/it/polimi/ingsw/model/Shelf.java
--- a/MyShelfie/src/main/java/it/polimi/ingsw/model/Shelf.java	
+++ b/MyShelfie/src/main/java/it/polimi/ingsw/model/Shelf.java	
@@ -59,7 +59,7 @@
     }
 
     /**Puts the assigned tiles in the coloumn having the assigned index*/
-    public void put_tiles (int col, ArrayList<Tile> tiles) {
+    public void put_tiles (int col, ArrayList<TileType> tiles) {
         for (int i = 5, j = 0; i >= 0 && j == 0; i--) {
             if (matrix[i][col].tile == null) {
                 for (int k = 0; k < tiles.size(); k++) {
@@ -72,17 +72,17 @@
 
 
     /**Returns the tile contained in the [x][y] shelf's slot*/
-    public Tile get_tile(int x, int y){
+    public TileType get_tile(int x, int y){
         return matrix[x][y].tile;
     }
 
     /**Returns the number of rows by which the Shelf is composed*/
-    public int n_row(){
+    public int nRow(){
         return 5;
     }
 
     /**Returns the number of coloumns by which the Shelf is composed*/
-    public int n_col(){
+    public int nCol(){
         return 6;
     }
 
